### ISSUE TITLE
Fix link to development packages

### ISSUE THREAD

**Issue by dustinsoftware** [reactjs/React.NET#825]
As a part of consolidating the CI services in #791 to use just Azure Pipelines, I missed that we are still linking to the [Appveyor package server](https://ci.appveyor.com/nuget/reactjs.net/). This issue discusses bringing that functionality back.

A few possibilities
- Publish alpha packages automatically to the main nuget feed any time there is a version bump in `build.proj`
- Push nuget packages to appveyor's nuget feed via a custom build step from azure pipelines ([docs](https://www.appveyor.com/docs/nuget/))
- Write some custom scripts to work around the msbuild issues in Appveyor

**Comment by dustinsoftware** [comment]
I managed to get package publishing working from Azure pipelines, although it is still a manual process to promote a build. That way several commits can land into master without resulting in a package publish for each CI build.

Under Azure Pipelines > Releases, click Create Release and pick the build to be published. We still need to bump the major/minor/patch parts of the version in `build.proj` before creating a release. I'd like to see if Azure pipelines can detect the change in version number and automatically publish a new release, which would remove an extra step...

Eg a bump from 5.0.0-alpha5 to 5.0.0-alpha6 should automatically publish a new package.

**Comment by Daniel15** [comment]
Really cool, thanks for looking into it 😄

**Comment by dustinsoftware** [comment]
Planning on moving to the Github package registry, but ran into a few blockers:

- Package publishing appears to be rate limited. Trying to use the nuget CLI to push all packages only results in the first few succeeding. Not sure how to get past this one without putting a sleep in the build, waiting to hear back from support on this one
- My personal access token allows pushing packages to my fork, but not the reactjs repo. This is a problem because the package source is specified at build time via the `projectUrl` or `repository` element in the generated `nuspec` from the build. We will likely need to just wait for the beta to be more widely opened.

The Azure pipelines deploy is [here](https://dev.azure.com/reactnet/ReactJS.NET/_releaseProgress?_a=release-environment-logs&releaseId=12&environmentId=12) and is all ready to go once the blockers are resolved.

**Comment by Daniel15** [comment]
We could try asking Github support about the rate limiting.

**Comment by dustinsoftware** [comment]
Yep, I sent them a support email about five minutes ago :) will see what they say

**Comment by Daniel15** [comment]
Did you hear back from GitHub about the rate limiting?

**Comment by dustinsoftware** [comment]
Yeah, when publishing from azure pipelines it isn’t a problem. So now we
just need the reactjs org to be enabled in the beta.
